### DISCUSSION TITLE
is possible monitor sycm_dir to specific dir of OneDrive?

### DISCUSSION THREAD

**Discussion by AtticusZeller** [abraunegg/onedrive#2948] [Q&A] [answered]
hey  there,👋 thank you for developing such useful client which does help me a lot ☺️

I check the document and search for hours on how to use OneDrive specific directory as sync endpoint?

for example, there are several dir on OneDrive but only the `Ubuntu` is the destination? 

make sure files or folders under local `snyc_dir` are identical with `OneDrive/Ubuntu`

**Comment by abraunegg** [comment, marked as answer]
Please read the advanced usage documentation - specifically:

https://github.com/abraunegg/onedrive/blob/master/docs/advanced-usage.md#upload-data-from-the-local-onedrive-folder-to-a-specific-location-on-onedrive

**Comment by AtticusZeller** [comment]
wow thank you so much bro!

Sent from Proton Mail Android

-------- Original Message --------
